src/cartesi-machine.lua

```diff
@@ -257,8 +257,80 @@ where options are:
   --auto-reset-uarch-state
     automatically reset state after miroarchitecture halt
 
-  -i or --htif-console-getchar
-    run in interactive mode.
+  --htif-console-getchar
```

Comment:
I think this option needs a better help message. 
It should be mutually exclusive with --virtio-console, right? What does it mean to use both? --htif-console-getchar needed for interactivity? (I.e., to use the use that trick that syncs with wall-clock time?) If so, I think we need to separate these concepts.